nuxeo-ai-core/src/main/java/org/nuxeo/ai/bulk/AbstractRecordWriter.java

```diff
@@ -89,8 +92,15 @@ public Optional<Blob> complete(String id) throws IOException {
                 if (managedBlob != null) {
                     return Optional.of(managedBlob);
                 }
+            } else {
+                log.warn("Unable to complete record {} as the file {} does not exist or is empty: {}", id, filename,
```

Comment:
Are these file names automatically generated by our code or are they provided by users of the system? Are there any concerns with PII in the logs from including the file name?